src/app/pages/server-details/server-details.component.ts

```diff
@@ -762,6 +773,24 @@ export class ServerDetailsComponent implements OnInit, OnDestroy {
       return undefined;
     }
 
+    if(name === 'storages') {
+      let html = '<ul>';
+      (prop as any[]).forEach((s: any, index: number) => {
+        html += `<li>${s.size}GB ${s.storage_type ? s.storage_type : ''}</li>`;
```

Comment:
there should be a space between the value and unit. same for `MB` below
```suggestion
        html += `<li>${s.size} GB ${s.storage_type ? s.storage_type : ''}</li>`;
```